### ISSUE TITLE
A.03.IR09.extension.for.QoS.declared.performance.md

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-interoperable-sds#8]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment
- The Prerequisites still references the now obsolete profile schema.

-The XPath still references the now obsolete profile schema.

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
